core/src/main/scripts/importer/libImportOncokb.py

```diff
@@ -0,0 +1,236 @@
+#!/usr/bin/env python3
+
+#
+# Copyright (c) 2020 The Hyve B.V.
+# This code is licensed under the GNU Affero General Public License (AGPL),
+# version 3, or (at your option) any later version.
+#
+
+#
+# This file is part of cBioPortal.
+#
+# cBioPortal is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+"""OncoKB annotation import script.
+
+Run with the command line option --help for usage information.
+"""
+
+import re
+import os
+import requests
+import json
+
+from . import validateData
+
+
+BATCH_SIZE = 200
+
+
+def load_portal_genes(server_url):
+    parsed_json = request_from_portal_api(server_url, "genes", logging.getLogger(__name__))
+    return transform_symbol_entrez_map(parsed_json, 'hugoGeneSymbol')
+
+
+# after: getProteinStart() in ExtendedMutationUtil.java
+def get_protein_pos_start(protein_position, protein_change):
+    start = -1
+    if protein_position is not None:
+        start_end = protein_position.split('/')[0].split('-')
+        if len(start_end) > 0:
+            start = start_end[0]
+    if start == -1 and protein_change is not None:
+        start_stop = annotate_protein_change(protein_change)
+        start = str(start_stop['start'])
+    return start
+
+
+# after: getProteinPosEnd() in ExtendedMutationUtil.java
+def get_protein_pos_end(protein_position, protein_change):
+    stop = -1
+    if protein_position is not None:
+        start_end = protein_position.split('/')[0].split('-')
```

Comment:
What's your normal protein position? In the Maf file we are using, it's annotate_protein_change  a single digit for the missense mutation. The protein end is start_end[0]

Reply:
I do not understand this comment. Can you clarify?

Reply:
Oh sorry, nvm. I just realize the line 70-72 solves my concern. My original thought is that in the maf file for missense mutation, we use just one number. For instance V600E, we use 600 as protein_position, rather 600-600. So based on the line 67, your `end` will always be -1 for missense mutation.